rfcs/README.md

```diff
@@ -0,0 +1,98 @@
+# oneDPL Design Documents/RFCs
+
+The Request for Comments (RFC) process intends to:
+
+- Propose and discuss ideas for the library evolution
```

Comment:
Please add periods to the end of each bullet point. In combination with the initial sentence, they are all complete sentences (unlike the list below with a single state).